editor/src/components/canvas/canvas-strategies/canvas-strategy-types.ts

```diff
@@ -13,13 +13,15 @@ export interface CustomStrategyState {
   escapeHatchActivated: boolean
   lastReorderIdx: number | null
   duplicatedElementNewUids: { [elementPath: string]: string }
+  updatedTargetPaths: Map<ElementPath, ElementPath>
```

Comment:
This will rely on the `ElementPath` instances being unique for a given path right?

Reply:
Hmmm yeah so we mostly ensure that, but actually I'll change this to use a stringified version for the key (since the `toString` function we created is fast anyway)